### ISSUE TITLE
Clarify licensing

### ISSUE THREAD

**Issue by chrysophylax** [ids/mule-cifs-connector#1]
Hi,

It would be good to clarify the license status of this code. Additionally, if it's derived from the https://github.com/mulesoft-consulting/samba-connector the MIT license blurb should be included.

PS. Nice job porting.

**Comment by seanhig** [comment]
You are quite right, please feel free to fork it and add the MIT back, this was mostly an oversight (and somewhat due to lack of interest in the platform).

I leave this repo up as a courtesy to others who may find themselves with a similar need, but it is not maintained. I avoid Mulesoft like the plague.  It is hideous.
